server/tests/audit_math/test_macro.py

```diff
@@ -208,6 +250,83 @@ def test_get_sample_sizes(contests, batches):
         )
 
 
+def test_full_recount(contests, batches):
+    # Do a full recount:
+    sample = batches
+    expected_recount_size = {
+        "Contest A": 1,
+        "Contest B": 1,
+        "Contest C": 1,
+    }
```

Comment:
why would a full recount only result in a sample size of one batch? i would expect the sample size to be the total number of batches

Reply:
This is default behavior for the sake of being conservative. In this test we're passing in our batches as the sample, so we've already done a full recount but we're asking for more ballots. I could change it to throw an error like I have done elsewhere.

Reply:
ah i get it. didn't see that we had already done the full recount. yes, i like the consistency of erroring here